.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "extends": "airbnb-base",
+  "rules": {
+    "camelcase": 0,
+    "comma-dangle": 0,
+    "dot-notation": 0,
+    "func-names": 0,
+    "no-console": 0,
+    "no-extra-semi": 0,
+    "no-global-assign": 0,
+    "no-redeclare": 0,
+    "no-shadow": 0,
+    "no-undef": 0,
```

Comment:
Also with this rule. As we have at least the global `Spree` object we could whitelist it here and keep the rule enabled.